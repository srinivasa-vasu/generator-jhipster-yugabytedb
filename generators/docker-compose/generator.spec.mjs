// import prom from 'expect';

// import { helpers, lookups } from '#test-utils';

// const SUB_GENERATOR = 'docker-compose';
// const BLUEPRINT_NAMESPACE = `jhipster:${SUB_GENERATOR}`;
// const { expect } = prom;

// describe('SubGenerator docker-compose of YugabyteDB JHipster blueprint', () => {
//   describe('run', () => {
//     let result;
//     before(async function () {
//       result = await helpers
//         .create(BLUEPRINT_NAMESPACE)
//         .withOptions({
//           reproducible: true,
//           defaults: true,
//           blueprint: 'yugabytedb',
//         })
//         .withLookups(lookups)
//         .run();
//     });

//     it('should succeed', () => {
//       expect(result.getStateSnapshot()).toMatchSnapshot();
//     });
//   });
// });
