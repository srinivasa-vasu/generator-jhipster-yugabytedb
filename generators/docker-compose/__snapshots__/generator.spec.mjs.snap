// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubGenerator docker-compose of YugabyteDB JHipster blueprint run should succeed 1`] = `
Object {
  "../0166eb07f9e1fc10c8d9335d40ebb63f54447e91/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../03df541f788de41fa5b43891034b726dc8007cbb/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../06a764cee9e7c5ac46a1cc860934d75aae71a156/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../0de9cff018017b490a2b52ecfe6ed4c2dbabc1eb/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../1199b5d156a3e6022a74168da16b10f2efb7f036/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../1747712722132e770b14ebe682a31098b01a45f8/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../207c320b42d73d0b22ddec29a8172d987253a736/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../211c999a0773c404b5ce39b3652b8ec41d18d5f6/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../22dffeb4d01c2d55fad34d0a9766230d201deaa8/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../23cfb9bb9cf49adba82251b99074e09b51dc0de3/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../246fdccd3b6608c1e18adbee18666ea7ce0c6598/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../295aa44af199300fcd14c82c7169b405665f6470/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../2a5353982a1eb03292c553f2d8405cc1c2643bc7/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../2b02299a3472773c736c5c0631bb835783ae9c6b/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../326feb52fa9b20def2850f29dfdaccb8208498a7/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../3565c7224900b29ce673ebc74637ad622ca8b404/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../381a188d6418ea6691eae11c4fad53a4470e1ecc/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../3989f46208dd80ab522bd62208a8abb273b0404c/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../416a021ccb3ebdcf0ab6e4e6d9f877b0f7bddd9e/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../434ae53fb3285630e83d233ed713918e07ec26a6/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../450f723b97bc9507bb21ac166b732103b5b53080/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../489c8ce2aa107e1688c8bfd538829bf1c6f875fd/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../4a12064f259e27a9f7a1786d83d297e1f9d3fd7b/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../4cda8adf2d7750ad600a2bdb5e74f0e65d5b2a2d/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../4e8fb1aeee39fb306890e2cba5da57cc6021bd34/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../507bbe354674ef61fef4ace43441c52635202738/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../549605c29c89a12040e4e82e036cc8febee2a610/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../55b55243a2371d103ec7e369a36ac95c05695ead/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../55f21e8d3ec0ad4c55e0814ddbb120a7db6e0593/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../5a8a7e8e9f71435e5bb76f8ceb1465ae8fa090fe/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../612da05063d651a889d52dcdb885bad7c35ca934/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../62903a12075b54ff24f33b0c0bda70aa2bb57f71/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../655d8970002ca6243ef6b692e4e0c3c2d68e83f2/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../683401c350b2adf8b8c6aad857256664430008fb/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../6daeaee1b85ac05e57e2bacd0a5e5db6610bd046/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../76cc08b716927449b10e7f815569c302d7a7e3cc/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../7846de539551543d0cc68a54c36fa3f0b8d08ea1/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../7cd853cfe44df541672c1a97c5b23f5aeec8cf72/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../86be6fd858b495ef515d17e2b8ef00051595cad8/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../88a26383ebe6edbc2cb38754e84c7c16b659c1c4/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../9bca22d9c96a374a9b52cac2756371406aedcf64/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../9c234df6bbf13cc303a7dce74ae8b0aa6ca866f7/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../a23cda479d30fc83370672a3bde2bf2181fca7b3/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../a465562a40f7d69d04a902f1424b874cc76ffb40/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../b017645dfad7dd4f32c028ace4e1f603d113f913/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../b6c30ad52820a251698fc011ff217d90adc0cbda/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../b6fd976b79ed59bb5d192ac699b33c1620b3cf2d/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../b7f34b7dcd189d29cd07592809389419cc66d339/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../bef5868c06cd117e762b59c58db8a650b579ef19/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../c14f8722e19cf91588e58270946167ed610cadda/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../c4e9e4db957ddefb377d6559600a496e6a5454bb/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../e3eb319ac5b7fd5a569baf07c643d389a777dd3a/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../e9899975dc8ee2b4901fe0055401a4d8df5d93cf/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../eb24992f77dd0f8e2b513e00a475a809013f815f/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../f1db3a9e370d62bf38aa459e790b2609f728e5ab/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "../f9a0cbd31c030cf4d4375f348dbcfd2fba1d6a03/.yo-rc.json": Object {
    "stateCleared": "modified",
  },
  ".yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "README-DOCKER-COMPOSE.md": Object {
    "stateCleared": "modified",
  },
  "docker-compose.yml": Object {
    "stateCleared": "modified",
  },
}
`;
